docs/tsconfig.json

```diff
@@ -10,6 +10,12 @@
     "esModuleInterop": true,
     "incremental": true
   },
-  "include": ["pages/**/*.ts*", "data/**/*", "src/modules/components/**/*", "next.config.mjs"],
+  "include": [
+    "pages/**/*.ts*",
+    "data/**/*",
+    "src/modules/components/**/*",
+    "next.config.mjs",
+    "../node_modules/@mui/monorepo/docs/types/*"
```

Comment:
I am not sure if this is the correct solution, but it fixed failing `test_types`: https://app.circleci.com/pipelines/github/mui/mui-x/63197/workflows/4f40377f-69e1-40ef-b8f1-5cec2a295908/jobs/360829

Reply:
Adding `@types/gtag.js` was not enough? 🤔 
Have you tried adding a `types: ["@types/gtag.js"]` into this file instead? 🤔

Reply:
> Adding @types/gtag.js was not enough? 🤔

I'm pretty sure it wasn't enough, but now it seems to be enough 😅